themes/custom/az_barrio/az_barrio.theme

```diff
@@ -35,7 +119,12 @@ function az_barrio_preprocess_html(&$variables) {
 /**
  * Implements hook_preprocess_HOOK() for page templates.
  */
-function az_barrio_preprocess_page(&$variables) {
+function az_barrio_preprocess_page(&$variables, $hook) {
+  $variables['az_barrio_navbar_offcanvas'] = (theme_get_setting('az_barrio_navbar_offcanvas')) ? TRUE : FALSE;
+  $variables['navbar_offcanvas_attributes'] = [
+    'class' => [ 'navbar-offcanvas' ],
+       'id' => 'navbar-top',
+  ];
   // Attach Arizona Bootstrap & Brand icon assets.
   $variables['#attached']['library'][] = 'az_barrio/arizona-bootstrap';
   $variables['#attached']['library'][] = 'az_barrio/az-brand-icons';
```

Comment:
Should this be removed since it's being handled in `hook_library_info_alter()` now?